packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -10,24 +8,14 @@ import {
     PseudoUrlInput,
 } from './shared';
 import { RequestQueue, QueueOperationInfo } from '../storages/request_queue';
-import { PseudoUrl } from '../pseudo_url';
 import { validators } from '../validators';
-import { CheerioRoot } from '../utils';
 
-export interface EnqueueLinksOptions {
+export interface BaseEnqueueLinksOptions {
```

Comment:
let's keep the interface name here as it was
```suggestion
export interface EnqueueLinksOptions {
```